src/HomePage.js

```diff
@@ -1,12 +1,47 @@
 import React from 'react';
 
```

Comment:
Once you pull and rebase the new template, you should have this line of code, which is necessary for your website to function properly: 
`import 'bootstrap/dist/css/bootstrap.min.css';`